### ISSUE TITLE
Can't type characters with Alt on Chrome

### ISSUE THREAD

**Issue by darthwiz** [akahuku/wasavi#192]
Hello,

using Chrome 64 and Wasavi 0.7.737 on OSX I'm having trouble inserting characters obtained with an Alt+key combination. For example, when I press Alt+è on an Italian keyboard, I should get an open square bracket, but nothing happens. If it can be of any use, I don't think the "Alt" keypress is missed, otherwise I would get the "è" character. Instead I get nothing, so perhaps the combination is correctly recognized.

**Comment by maronk** [comment]
I can confirm this is happening on my machine too.  Icelandic keyboard layout.

**Comment by maronk** [comment]
@darthwiz Did  you find any solution to this, this is making a highly usable plugin unusable to us

**Comment by darthwiz** [comment]
@maronk nope, sorry - I gave up after a few days, which is really sad because if it weren't for this problem it would make github editing really powerful. I hope the author is tuned in and interested in fixing it.

**Comment by maronk** [comment]
I read about a similar issue in the past and took to: http://appsweets.net/wasavi/qeema/ 

These are the codes I'm currently having problem with  Alt 7,8,9,0 - output on an Icelandic keyboard should be {[]} - as a programmer I'm having a hard time using the plugin

[code:  -8193] [char:                 ] [key: <A-{>           ] [modifier: A--] [isSpecial: false] [composition: no]
[code:  -8193] [char:                 ] [key: <A-[>           ] [modifier: A--] [isSpecial: false] [composition: no]
[code:  -8193] [char:                 ] [key: <A-]>           ] [modifier: A--] [isSpecial: false] [composition: no]
[code:  -8193] [char:                 ] [key: <A-}>           ] [modifier: A--] [isSpecial: false] [composition: no]

I'm using MacOS High Sierra

**Comment by maronk** [comment]
Might be related with https://github.com/akahuku/wasavi/issues/158 and https://github.com/akahuku/wasavi/issues/47
